src/components/List/List.jsx

```diff
@@ -5,20 +5,45 @@ import { useFirebase } from '../../hooks/useFirebase';
 const List = () => {
   const token = localStorage.getItem('token');
 
-  const { getAll } = useFirebase();
+  const { getAll, update } = useFirebase();
 
   const firebasePath = getAll().doc(token).collection('items');
 
   const [value, loading, error] = useCollection(firebasePath);
 
+  const handleCheck = (id, lastDate) => {
+    if (isPassed24Hours(lastDate) === false) {
+      const date = new Date();
+      update(token, id, { lastDate: date });
+    }
+  };
+
+  const isPassed24Hours = (date) => {
```

Comment:
Small nit-picky grammar thing: `isPast24Hours` or `has24HoursPassed`

Reply:
thanks for the correction Mike, the change has been done 👍